docs/docs/API-Reference/api-openai-responses.mdx

```diff
@@ -22,6 +25,82 @@ The component types `ChatInput` and `Chat Input` are recognized as chat inputs.
 - **Authentication**: All requests require an API key passed in the `x-api-key` header.
 For more information, see [API keys and authentication](/api-keys-and-authentication).
 
+### Using OpenAI Client Libraries
+
+This endpoint is compatible with OpenAI's API, but requires special configuration when using OpenAI client libraries.
+Langflow uses `x-api-key` headers for authentication, while OpenAI uses `Authorization: Bearer` headers.
+When sending requests to Langflow with OpenAI client libraries, you must configure custom headers and include an `api_key` configuration.
+The `api_key` parameter can have any value, such as `"api-dummy-key"` in the client examples, as the actual authentication is handled through the `default_headers` configuration.
+
+Replace the values for `LANGFLOW_SERVER_URL`, `LANGFLOW_API_KEY`, and `FLOW_ID` with values from your deployment.
```

Comment:
```suggestion
In the following examples, replace the values for `LANGFLOW_SERVER_URL`, `LANGFLOW_API_KEY`, and `FLOW_ID` with values from your deployment.
```

Alternatively, you can move this line (in its original, unedited form) below the tabs or inside each tab.